examples/04-write-to-persistent/client.c

```diff
@@ -243,6 +243,13 @@ main(int argc, char *argv[])
 	if (ret)
 		goto err_conn_disconnect;
 
+	/* wait for the completion to be ready */
+	ret = rpma_conn_prepare_completions(conn);
```

Comment:
@janekmi 
What's the reason that CQE isn't posted into CQ since we've posted the read WR into SQ?
BTW, ibv_ack_cq_events is said to be time-consuming.
What's about below changes:
```c
diff --git a/src/conn.c b/src/conn.c
index 3e227af..2b18e21 100644
--- a/src/conn.c
+++ b/src/conn.c
@@ -391,7 +391,12 @@ rpma_conn_next_completion(struct rpma_conn *conn, struct rpma_completion *cmpl)
                /*
                 * There may be an extra CQ event with no completion in the CQ.
                 */
-               return RPMA_E_NO_COMPLETION;
+               if (rpma_conn_prepare_completions(conn) == 0) {
+                       result = ibv_poll_cq(conn->cq, 1 /* num_entries */, &wc);
+               }
+               if (result == 0) {
+                       return RPMA_E_NO_COMPLETION;
+                }
        } else if (result < 0) {
                Rpma_provider_error = result;
                return RPMA_E_PROVIDER;
```